TransIT.BLL/Helpers/Warranties.cs

```diff
@@ -1,8 +1,9 @@
 ﻿namespace TransIT.BLL.Helpers
 {
-    public static class Warranties
+    // needs to investigate if this enum is realy needed
+    public class Warranties
     {
```

Comment:
- remove this class and in usages just set 'true' or 'false', eg. issue.Warranty = true;